packages/lib/src/wrappers.ts

```diff
@@ -10,7 +19,6 @@ import {
   isObject,
   isPromise,
 } from "./utils";
-import { setBuildInfo } from "./buildInfo";
```

Comment:
Do you use Rome's import organizer? Curious what's your experience if so :)

Reply:
This was actually `tsserver`'s own API for organizing imports! :)